### DISCUSSION TITLE
rss and tags: is it possible a feed for every tag with annotations for every Item ?

### DISCUSSION THREAD

**Discussion by iusondemand** [wallabag/wallabag#6585] [General]
First of all, sorry for my english.

I use the feeds generated by tags. Example: https://app.wallabag.it/feed/user/randomlink/tags/mytags

I would like to receive also the annotations I take manually (very powerfull feature). Rss doesn't returns annotations.

Perhaps it's possible to add annotations to the rss feeds of every tag with <annotations></annotations> tags; or perhaps we could support a separate json version with an annotations array.

Is it possible ? I'm a developer, perhaps I could help, but I don't know well the code. Where's the function that builds the rss feed for each tag ? 

Thank you

**Comment by iusondemand** [comment]
Sorry for mistakes. I correct the question:

"I would like to receive also the annotations . Rss doesn't return annotations."

"Perhaps it's possible to add annotations to the rss feeds; or perhaps we could support a separate json version with an annotations array."

**Reply by nicosomb** [reply]
Hello @iusondemand. 

I don't know how we can handle annotations in the rss feed... 
I don't know if RSS specs allow this kind of features.

**Reply by iusondemand** [reply]
You're right, but in podcasts it is so usual to find images that I didn't know the limit.
So this is the official way, add this line in every item:

<itunes:image href="http://www.privacykit.it/images/logoprivacykit3000.jpg"/>

Please note that itunes is a standard supported by every player.

This is an example:
https://www.privacykit.it/podcast/rss.php

Some posts on this matter:
- historically, no standards for images: https://podcastpage.io/exploring-the-new-standard-for-podcast-rss-feeds/
- a recent proposal, with some support : https://podcastindex.org/namespace/1.0#images

In my experience (since 2008) itunes:image tag is widely supported by every podcast player.

Let me know if you need more infos

Thanks !
